website/static/js/home-page/quickProjectSearchPlugin.js

```diff
@@ -46,7 +47,7 @@ var QuickSearchProject = {
         self.templateNodes.start();
 
         // Load up to first ten nodes
-        var url = $osf.apiV2Url('users/me/nodes/', { query : { 'embed': 'contributors'}});
+        var url = $osf.apiV2Url('users/me/nodes/', { query : { 'embed': ['contributors','root', 'parent']}});
```

Comment:
Ah, right, because of the searching. Well, worth profiling, I guess, although it shouldn't affect the UX.